jni/tests/main.cpp

```diff
@@ -29,6 +26,12 @@
 #include "processors/tremolo_test.cpp"
 #include "utilities/fastmath_test.cpp"
 #include "utilities/tablepool_test.cpp"
+#include "deprecation_test.cpp"
+
+// these aren't stability tests, but benchmarks to test certain performance assumptions
+//#include "benchmarks/buffer_test.cpp"
+//#include "benchmarks/inline_test.cpp"
+//#include "benchmarks/table_test.cpp"
 
```

Comment:
These fail randomly and across devices, need to follow up to get the proper behaviour depending upon consistent benchmarks